var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -36,7 +36,7 @@ class Coreneuron(CMakePackage):
     homepage = "https://github.com/BlueBrain/CoreNeuron"
     url      = "https://github.com/BlueBrain/CoreNeuron"
```

Comment:
add `git = …` here, remove all other `git=url` from file!

Reply:
Good point.. I will add this to the `neuron cmake` PR which I have open to avoid opening another PR only for this.

Reply:
Awesome, thanks!